fdbserver/Status.actor.cpp

```diff
@@ -1928,7 +1928,7 @@ ACTOR static Future<std::vector<std::pair<StorageServerInterface, EventMap>>> ge
 	                            std::vector<std::string>{
 	                                "StorageMetrics", "ReadLatencyMetrics", "ReadLatencyBands", "BusiestReadTag" })) &&
 	     store(busiestWriteTags, getServerBusiestWriteTags(servers, address_workers, rkWorker)) &&
-	     store(metadata, getServerMetadata(servers, cx, true)));
+	     store(metadata, timeoutError(getServerMetadata(servers, cx, true), 5.0)));
```

Comment:
```suggestion
    wait(timeoutError(
	    store(results,
	          getServerMetrics(servers,
	                           address_workers,
	                           std::vector<std::string>{
	                               "StorageMetrics", "ReadLatencyMetrics", "ReadLatencyBands", "BusiestReadTag" })) &&
	        store(busiestWriteTags, getServerBusiestWriteTags(servers, address_workers, rkWorker)) &&
	        store(metadata, getServerMetadata(servers, cx, true)),
	    5.0));
```

Reply:
Both `getServerMetrics` and `getServerBusiestWriteTags` invoke `latestEventOnWorker`, which already has timeout set. So only `getServerMetadata` need the change here.

Reply:
Why we don't need to consider the timeout issue for the other two actors here?

Reply:
Because they internally has timeout set, and will finish within bounded time.